lib/geocoder/providers/google_maps.ex

```diff
@@ -5,8 +5,10 @@ defmodule Geocoder.Providers.GoogleMaps do
   @endpoint "https://maps.googleapis.com/"
 
   def geocode(opts) do
-    request("maps/api/geocode/json", extract_opts(opts))
-    |> fmap(&parse_geocode/1)
+    case request("maps/api/geocode/json", extract_opts(opts)) do
+      {:error, reason} -> error(reason)
```

Comment:
I know this lib already uses [towel](https://github.com/knrz/towel) and its `ok` / `error` wrappers. However, we're explicitly returning :error and :ok tuples already so I would prefer removing the level of indirection `ok()` and `error()` introduces here.